Assets/MixedRealityToolkit/_Core/Definitions/InteractionDefinition.cs

```diff
@@ -23,5 +30,141 @@ public struct InteractionDefinition
         /// The primary action of the button as defined by the controller SDK.
         /// </summary>
         public InputType InputType { get; set; }
+
+        #endregion Interaction Properties
+
+        #region Definition Datums
+
+        /// <summary>
+        /// The data storage for a Raw / None Axis type.
+        /// </summary>
+        private object rawDatum { get; set; }
```

Comment:
Let's drop the datum suffix on these properties and use data instead. Datum is plural for data and we only have a single item.

Reply:
Fair enough, got to get our connotations right.